bookwyrm_frontend/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:lts-alpine
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+EXPOSE 8080
```

Comment:
If I'm understanding this correctly, I believe this should be 80, since the front end runs on port 80.

Reply:
I have changed the port to 80. However, as long as you map to the right port of the docker container, the app works.